release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3351,6 +3351,72 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the Cluster ingress operator logged non-existent updates. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-39324[*OCPBUGS-39324*])
+
+* Previously, the installation program failed to install an {product-title} cluster in the `eu-es` (Madrid, Spain) region on a {ibm-power-server-title} platform that was configured as an `e980` system type. With this release, the installation program no longer fails to install a cluster in this environment. (link:https://issues.redhat.com/browse/OCPBUGS-38502[*OCPBUGS-38502*])
+
+* Previously, the ingress controller `Degraded` status would not set because of an issue with the `CanaryRepetitiveFailures` condition transition time. With this release, the condition transition time is only updated when the condition status changes, instead of when the message or reason are the only changes. (link:https://issues.redhat.com/browse/OCPBUGS-39323[*OCPBUGS-39323*])
+
+* Previously, an `AdditionalTrustedCA` field that was specified in the Hosted Cluster image configuration was not reconciled into the openshift-config namespace as expected and the component was not available. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-39293[*OCPBUGS-39293*])
+
+* Previously, an installer regression issue caused problems with Nutanix cluster deployments using the Dynamic Host Configuration Protocol (DHCP) network. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38956[*OCPBUGS-38956*])
+
+* Previously, a rare condition caused the CAPV session to time out unexpectedly. With this release, the *Keep Alive* support is disabled in later versions of CAPV and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38822[*OCPBUGS-38822*])
+
+* Previously, the version number text in the updates graph on the Cluster Settings appeared as black text on a dark background while viewing the page using Firefox in dark mode. With this update, the text appears as white text. (link:https://issues.redhat.com/browse/OCPBUGS-38822[*OCPBUGS-38424*])
+
+* Previously, proxying for Operators that run in the control plane of a Hypershift cluster was performed through proxy settings on the konnectivity agent pod that runs in the data plane. As a result, it was not possible to distinguish whether proxying was needed based on application protocol. For parity with {rh-short}, IDP communication through https/http should be proxied, but LDAP communication should not be proxied. With this release, how proxy is handled in hosted clusters is changed to invoke the proxy in the control plane via konnectivity-https-proxy and konnectivity-socks5-proxy, and to stop proxying traffic from the konnectivity agent. (link:https://issues.redhat.com/browse/OCPBUGS-38062[*OCPBUGS-38062*])
```

Comment:
```suggestion
* Previously, proxying for Operators that run in the control plane of a HyperShift cluster was performed through proxy settings on the konnectivity agent pod that runs in the data plane. As a result, it was not possible to distinguish whether proxying was needed based on application protocol. For parity with {rh-short}, IDP communication through https/http should be proxied, but LDAP communication should not be proxied. With this release, how proxy is handled in hosted clusters is changed to invoke the proxy in the control plane via `konnectivity-https-proxy` and `konnectivity-socks5-proxy`, and to stop proxying traffic from the konnectivity agent. (link:https://issues.redhat.com/browse/OCPBUGS-38062[*OCPBUGS-38062*])
```

Reply:
- Searching the repo, I think HyperShift the S is capitalized.
- do these `konnectivities` need backticks? It seems like it to me